.ado/windows-vs-pr.yml

```diff
@@ -26,30 +26,31 @@ jobs:
     dependsOn: Setup
     condition: ne( dependencies.Setup.outputs['checkPayload.shouldSkipPRBuild'], 'True' )
     strategy:
-      matrix:
-        X64Debug:
-          BuildConfiguration: Debug
+      matrix: # Why we only build some flavors: https://github.com/microsoft/react-native-windows/issues/4308
+        #X64Debug:
+        #  BuildConfiguration: Debug
+        #  BuildPlatform: x64
+        X64Release:
+          BuildConfiguration: Release
           BuildPlatform: x64
-        #  LayoutHeaders: true
-        #X64Release:
+        X86Debug:
+          BuildConfiguration: Debug
+          BuildPlatform: x86
+        #X86Release:
         #  BuildConfiguration: Release
-        #  BuildPlatform: x64
+        #  BuildPlatform: x86
         #ArmDebug:
         #  BuildConfiguration: Debug
-        #  BuildPlatform: arm
-        #X86Debug:
-        #  BuildConfiguration: Debug
-        #  BuildPlatform: x86
-        X86Release:
-          BuildConfiguration: Release
-          BuildPlatform: x86
-        # Temporarily omitting to save disk space.
-        # ArmRelease:
-        #   BuildConfiguration: Release
-        #   BuildPlatform: arm
-        Arm64Release:
+        #  BuildPlatform: ARM
+        ArmRelease:
           BuildConfiguration: Release
-          BuildPlatform: arm64
+          BuildPlatform: ARM
+        #Arm64Debug:
+        #  BuildConfiguration: Debug
+        #  BuildPlatform: ARM64
+        #Arm64Release:
+        #  BuildConfiguration: Release
+        #  BuildPlatform: ARM64
```

Comment:
Realizing this is sub-optimal to not build all flavors for all apps, we can get still get pretty coverage by spreading the build flavors across the different test apps.  It looks like we're not building anything in arm64 flavor.  Is that still not working yet?  If not then it's OK to tackle that later.  The thought was have at least one arm64chk and arm64fre combo across some set of test apps.

Reply:
We have to retarget our min windows version before we cane have arm64 apps. #4031